studio/pages/integrations/github/install.tsx

```diff
@@ -48,18 +49,27 @@ const GitHubIntegration: NextPageWithLayout = () => {
     },
   })
 
+  const onInstalled = useCallback(
+    (id: string, orgSlug?: string) => {
+      setOrganizationIntegrationId(id)
```

Comment:
do we need this? just wondering cause we're changing routes as part of this - the state will get lost either way